commit/merkleroot/rmn/types.go

```diff
@@ -0,0 +1,36 @@
+package rmn
+
+import "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+type NodeSignatures struct {
```

Comment:
Wouldn't it just be better to use the proto types directly? Most of these types are identical to whats in the protos right?

Reply:
I dislike these custom types because I'm not sure there's much benefit. You'll have to manually edit them if/when the protos change, and for the `Client` interface, you can just use the proto type there as well directly and all will be fine when mocking

Reply:
Another argument: you have to translate from this type to the proto anyway, before marshaling the protos. So this just adds an unnecessary step to the process

Reply:
Yeah agree, can replace them while working on
> The implementation of this interface and .proto types generation.

Reply:
Where are these protos?
1. I agree that we should avoid copying types.
2. What about the [crypto/ecdsa](https://pkg.go.dev/crypto/ecdsa) package?